### ISSUE TITLE
WDK record not found sometimes throws 500

### ISSUE THREAD

**Issue by aurreco-uga** [VEuPathDB/EbrcModelCommon#46]
(found URL in error emails)
check teh difference between:
https://microbiomedb.org/mbio/app/record/dataset/DS_1c4b2eeecb
and
https://microbiomedb.org/mbio/app/record/dataset/bbbbbbbb

**Comment by aurreco-uga** [comment]
we need the aliasqueryref..

**Comment by ryanrdoherty** [comment]
@aurreco-uga No, that's not what I was saying.  Not sure why mbio datasets are trying to join to DatasetAttributes.ClinepiStudyAttributes, but that is the problem.
